website/src/controllers/api-v3/members.js

```diff
@@ -231,4 +236,53 @@ api.getChallengeMemberProgress = {
   },
 };
 
+/**
+ * @api {posts} /send-private-message Get a challenge member progress
+ * @apiVersion 3.0.0
+ * @apiName SendPrivateMessage
+ * @apiGroup Members
+ *
+ * @apiParam {String} message The message
+ * @apiParam {UUID} toUserId The toUser _id
+ *
+ * @apiSuccess {} Object Returns an empty object
+ */
+api.sendPrivateMessage = {
+  method: 'POST',
+  url: '/send-private-message',
```

Comment:
I think the route here should be under /members -> /members/send-private-message